app/main.js

```diff
@@ -122,10 +148,17 @@ function shutdownDaemon(evenIfNotStartedByApp = false) {
       console.log('Killed lbrynet-daemon process');
     });
   } else if (evenIfNotStartedByApp) {
-    console.log('Killing lbrynet-daemon, even though app did not start it');
-    client.request('daemon_stop', []);
-    // TODO: If the daemon errors or times out when we make this request, find
-    // the process and force quit it.
+    console.log('Stopping lbrynet-daemon, even though app did not start it');
+    client.request('daemon_stop', [], (err, res) => {
+      if (err) {
+        // We could get an error because the daemon is already stopped (good)
```

Comment:
Sorry, should have written "frozen," not "stopped." The daemon process could be open but still it's refusing connections, or timing out, or giving 500 errors. Even if the error is "connection refused," it could be frozen but still have the port bound.